universal-login-relayer/lib/utils/errors.ts

```diff
@@ -33,6 +33,13 @@ export class InvalidContract extends ValidationFailed {
   }
 }
 
+export class InvalidProxy extends ValidationFailed {
+  constructor (proxyHash: string) {
+    super(`Invalid proxy hash: ${proxyHash}`, 'InvalidSignature');
```

Comment:
Could be more descriptive, eg.:
`Invalid proxy at address ${proxyAddress}. Contract bytecode hash: ${proxyHash}. Supported hashes: [...]`

Reply:
✔️